src/opm/input/eclipse/Schedule/HandlerContext.hpp

```diff
@@ -80,15 +88,68 @@ class HandlerContext
     //! \brief Mark that well structure has changed.
     void record_well_structure_change();
 
+    //! \brief Returns a reference to current state.
+    ScheduleState& state();
+
+    //! \brief Returns a const-ref to the static schedule.
+    const ScheduleStatic& sched_stat() const;
```

Comment:
I think I'd prefer a **slightly** different name here.  `_stat()` **could** be misconstrued as `state`, so maybe something along the lines of `static_schedule()` or `initial_schedule()` or `starting_schedule()` could be an alternative?  There's not a lot to go on in PR #2206, but those names do at least match my impression of what this thing is.